src/framework/filterControl/filterDropdown/CreateFilterFormDropdown.jsx

```diff
@@ -0,0 +1,47 @@
+
```

Comment:
Also, the class names are already `filterFormDropdown` I would rename to `filterDropdownForm` instead because it is a filter dropdown for which there is a form.